### DISCUSSION TITLE
Unity Add Package from git URL failure.

### DISCUSSION THREAD

**Discussion by Geethanimal** [readyplayerme/rpm-unity-sdk-core#55] [Q&A]
I tried to add this package to unity using [https://github.com/readyplayerme/rpm-unity-sdk-core.git#v1.0.0 ](https://github.com/readyplayerme/rpm-unity-sdk-core.git#v1.0.0 ) this URL that provided by official documentation. But an error occurs.

The error is below. 

[Package Manager Window] Error adding package: https://github.com/readyplayerme/rpm-unity-sdk-core.git#v1.0.0.
Unable to add package [https://github.com/readyplayerme/rpm-unity-sdk-core.git#v1.0.0]:
  Error when executing git command. fatal: update_ref failed for ref 'HEAD': cannot update ref 'refs/heads/master': trying to write non-commit object c6abc7525bab6ab7f2b3e7e558d12789c8c3788f to branch 'refs/heads/master'
  
 I also used this URL [https://github.com/readyplayerme/Unity-core.git](https://github.com/readyplayerme/Unity-core.git) that is provided in the README of the repo but it takes a long time and not working.
  
 How can I resolve this error and install this package in unity?

**Comment by srcnalt** [comment]
Hi @Geethanimal have you tried installing the latest through `https://github.com/readyplayerme/rpm-unity-sdk-core.git`
Is the issue still present?

**Reply by qq235274300** [reply]
i use this url, takes a long time and not working too : (

**Reply by srcnalt** [reply]
Could you post if you are getting any error messages? And also could you update your git version just to be sure?

**Comment by Emmet1119** [comment]
> I tried to add this package to unity using [https://github.com/readyplayerme/rpm-unity-sdk-core.git#v1.0.0 ](https://github.com/readyplayerme/rpm-unity-sdk-core.git#v1.0.0) this URL that provided by official documentation. But an error occurs.
> 
> The error is below.
> 
> [Package Manager Window] Error adding package: https://github.com/readyplayerme/rpm-unity-sdk-core.git#v1.0.0. Unable to add package [https://github.com/readyplayerme/rpm-unity-sdk-core.git#v1.0.0]: Error when executing git command. fatal: update_ref failed for ref 'HEAD': cannot update ref 'refs/heads/master': trying to write non-commit object c6abc7525bab6ab7f2b3e7e558d12789c8c3788f to branch 'refs/heads/master'
> 
> I also used this URL https://github.com/readyplayerme/Unity-core.git that is provided in the README of the repo but it takes a long time and not working.
> 
> How can I resolve this error and install this package in unity?

I have gotten the exact same error
